components/services/bat_ledger/public/interfaces/bat_ledger.mojom

```diff
@@ -63,7 +63,7 @@ interface BatLedger {
       int32 month, int32 year, uint32 data);
 
   FetchGrants(string lang, string payment_id);
-  GetGrantCaptcha(array<string> headers);
+  GetGrantCaptcha(array<string> headers) =>(string image, string hint);
```

Comment:
Nit: Should have a space after the arrow.

Reply:
fixed